doc/flame/examples/lib/rotate_around_effect.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:doc_flame_examples/flower.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/game.dart';
+import 'package:flame/geometry.dart';
+
+class RotateAroundEffectGame extends FlameGame {
+  bool reset = false;
+  @override
+  Future<void> onLoad() async {
+    final flower = Flower(
+      size: 60,
+      position: canvasSize / 4,
+      onTap: (flower) {
+        if (reset = !reset) {
+          flower.add(
+            RotateAroundEffect(
+              tau,
```

Comment:
nit: how about moving the branching path only in the part that changes (1 vs -1) to avoid duplicating the rest?

Reply:
Copied this from the other one, I just realized how dirty this is.
Look at the assignment in the if... :sweat_smile:

Reply:
I'll clean them up

Reply:
I noticed, I thought it was the reason you didn't wanna use a ternary :P